[BE] Validate an user for a job by job owner

nspc97 opened the pull request:
[BE] Validate an user for a job by job owner
https://trello.com/c/Ik5lwLAp/60-be-validate-an-user-for-a-job-by-job-owner

Commit:
[BE] Validate an user for a job by job owner
https://trello.com/c/Ik5lwLAp/60-be-validate-an-user-for-a-job-by-job-owner

MihaiDruta commented on src/Controller/ActivityController.php:
```diff
@@ -684,4 +684,78 @@ public function listOfApplicants(Activity $activity, ActivityUserRepository $act
+     *     response="200",
+     *     description="Successfull operation!",
+     *     @SWG\Schema(
+     *     @SWG\Property(property="message", type="string", example="User invited with succes!"),
```
Wrong message. Should be "User assigned with succes!".

MihaiDruta commented on src/Controller/ActivityController.php:
```diff
@@ -684,4 +684,78 @@ public function listOfApplicants(Activity $activity, ActivityUserRepository $act
+
+        $userApplier = $activityUserRepo->isUserApplier($user, $activity);
+        if ($userApplier !== null) {
+            return new JsonResponse(['message' => 'User cannot be assigned!'], Response::HTTP_OK);
```
Wrong response code. Should be probably 400.

Commit:
[BE] Validate an user for a job by job owner
https://trello.com/c/Ik5lwLAp/60-be-validate-an-user-for-a-job-by-job-owner

Fixed response code

SpartakusMd commented on src/Controller/ActivityController.php:
```diff
@@ -684,4 +684,78 @@ public function listOfApplicants(Activity $activity, ActivityUserRepository $act
+        if ($userApplier !== null) {
+            return new JsonResponse(['message' => 'User cannot be assigned!'], Response::HTTP_BAD_REQUEST);
+        }
+        $activityUserRepo->assign($userApplier);
```
Method is redundant. Add it's contents here and remove it.

Commit:
[BE] Validate an user for a job by job owner
https://trello.com/c/Ik5lwLAp/60-be-validate-an-user-for-a-job-by-job-owner
